mettle/worker/worker.go

```diff
@@ -395,6 +395,7 @@ type worker struct {
 
 // ShutdownQueues shuts down the internal topic & subscription when done.
 func (w *worker) ShutdownQueues() {
+	log.Notice("Shutting down queues")
```

Comment:
You could include the action digest in here so we can see if it's shutting down with an in progress action